### DISCUSSION TITLE
arm64_v8The container status is not displayed

### DISCUSSION THREAD

**Discussion by chaomarks** [nicolargo/glances#2808] [General]
The command：
docker run -d -it \
--restart="always" \
--name=glances \
-p 61208-61209:61208-61209 \
-e TZ="${TZ}" \
-e GLANCES_OPT="-w" \
--cap-add=SYS_PTRACE  \
-v /var/run/docker.sock:/var/run/docker.sock:ro \
-v /run/user/1000/podman/podman.sock:/run/user/1000/podman/podman.sock:ro \
--pid host  \
nicolargo/glances:4.0.7-full
![image](https://github.com/nicolargo/glances/assets/93130581/cd71ccd1-d6ad-45d5-aa3a-778f54655c75)

**Comment by nicolargo** [comment]
Can you try to run Glances with the --privileged docker option ?
